### PR TITLE
AAP-4830 added new line to assembly file, and added new proc doc to m…

### PR DESCRIPTION
**Pull request by rogrange**
Adding new section, attempted to build, had failure.

### CODE REVIEW

**Comment by rogrange** [comment]
Added content and files for https://issues.redhat.com/browse/AAP-4830.

File: downstream/modules/platform/ref-converting-playbook-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+----
+[
+"/mydata:/mydata:rw"
+]
+----
+
+The volume mount is mapped with the same name in the container and has read-write capability. This will get used when you launch the job template.
+
+The _prompt on launch_ should be set for _extra_vars_ so you can adjust the sleep duration for each run, The default is 30 seconds.
```
Comment (sayjadha) [inline downstream/modules/platform/ref-converting-playbook-examples.adoc:73]: Typo, insert period here.

File: downstream/modules/platform/ref-converting-playbook-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+To verify the run has completed successfully, run this command to get an output of the job:
+
+----
+$ podman exec -it 'podman ps -q' /bin/bash
+bash-4.4#
+----
+
+You are now inside the running execution environment container.
+
+Look at the permissions, you will see that *awx* has become ‘root’, but this is not really root as in the superuser, as you are using rootless Podman, which maps users into a kernel namespace similar to a sandbox. Learn more about How does rootless Podman work? for shadow-utils.
```
Comment (sayjadha) [inline downstream/modules/platform/ref-converting-playbook-examples.adoc:86]: Do we need a cross-reference here?

File: downstream/modules/platform/ref-converting-playbook-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+...ignoring
+
+TASK [Read written out file...] ****************************** 10:50:13
+Fatal: [localhost]: FAILED: => {"msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError;>, original message: could not locate file in lookup: /mydate/file_write. Vould not locate file in lookup: /mydate/file_write"}
+...ignoring
+----
+
+The job failed, even though *:rw* is set, so it should have write capability. 
+The process was able to read the existing file, but not write out. This is due to SELinux protection that requires proper labels to be placed on the volume content mounted into the container. If the label is missing, SELinux may prevent the process from running inside the container. Labels set by the OS are not changed by Podman. See the Podman documentation for more information.
```
Comment (sayjadha) [inline downstream/modules/platform/ref-converting-playbook-examples.adoc:116]: Do we need a cross-reference to the Podman docs here?

File: downstream/modules/platform/proc-install-ansible-colls.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+= Installing Ansible collections
+
+
+
+As part of the migration from Ansible 2.9 to more recent versions, you need to find and download the collections that include the modules you have been using. Once you find that list of collections, you can use one of the following options to include your collections locally:
+
+. Download and install the Collection into your runtime or execution environments using `ansible-builder`.
+
+. Update the _requirements.yml_ file in your Automation Controller project install roles and collections. This way every time you sync the project in Automation Controller the roles and collections will be downloaded.
```
Comment (sayjadha) [inline downstream/modules/platform/proc-install-ansible-colls.adoc:13]: Highlight file names in monospace formatting. Ref: IBM style guide > 'Highlighting by element' section.

**Review by sayjadha — COMMENTED** [review]
Very minor edits, rest LGTM.

File: downstream/modules/platform/ref-converting-playbook-examples.adoc (hunk 2 above)
Comment (rogrange) [reply downstream/modules/platform/ref-converting-playbook-examples.adoc:86]: Yes and no. This reference is in the reference module.  We do not need an xref due to this being and example of what can happen when running jobs under SELinux.

File: downstream/modules/platform/ref-converting-playbook-examples.adoc (hunk 1 above)
Comment (rogrange) [reply downstream/modules/platform/ref-converting-playbook-examples.adoc:73]: Period has been added. Thanks!

File: downstream/modules/platform/ref-converting-playbook-examples.adoc (hunk 3 above)
Comment (rogrange) [reply downstream/modules/platform/ref-converting-playbook-examples.adoc:116]: Yes, and no. These are also referenced in the Reference module so as not to stunt the flow. The SME thought it was better to attempt to have the links elsewhere.

File: downstream/modules/platform/proc-install-ansible-colls.adoc (hunk 4 above)
Comment (rogrange) [reply downstream/modules/platform/proc-install-ansible-colls.adoc]: Formatting changed.

**Review by sayjadha — COMMENTED** [review]
LGTM.

**Pull request merged**